internal/conf/configuration.go

```diff
@@ -414,6 +416,59 @@ type WebhookConfig struct {
 	Events     []string `json:"events"`
 }
 
+// Moving away from the existing HookConfig so we can get a fresh start.
+type HookConfiguration struct {
+	MFAVerificationAttempt ExtensibilityPointConfiguration `json:"mfa_verification_attempt" split_words:"true"`
+}
+
+type ExtensibilityPointConfiguration struct {
+	URI      string `json:"uri"`
+	Enabled  bool   `json:"enabled"`
+	HookName string `json:"hook_name"`
+}
+
+func (h *HookConfiguration) Validate() error {
+	points := []ExtensibilityPointConfiguration{
+		h.MFAVerificationAttempt,
+	}
+	for _, point := range points {
+		if err := point.ValidateAndPopulateExtensibilityPoint(); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func (e *ExtensibilityPointConfiguration) ValidateAndPopulateExtensibilityPoint() error {
+	if e.URI != "" {
+		regExp := `^[a-zA-Z_][a-zA-Z0-9_]{0,62}$`
+		re, err := regexp.Compile(regExp)
+		if err != nil {
+			return err
+		}
+
+		u, err := url.Parse(e.URI)
+		if err != nil {
+			return err
+		}
+		pathParts := strings.Split(u.Path, "/")
+		if len(pathParts) < 3 {
+			return fmt.Errorf("URI path does not contain enough parts")
+		}
+		schema := pathParts[1]
+		table := pathParts[2]
+		// Validate schema and table names
+		if !re.MatchString(schema) {
+			return fmt.Errorf("invalid schema name: %s", schema)
+		}
+		if !re.MatchString(table) {
+			return fmt.Errorf("invalid table name: %s", table)
+		}
+		e.HookName = fmt.Sprintf("%s.%s", schema, table)
```

Comment:
You probably mean to use this here instead. Imagine a URL like this:

```
pg-functions://postgres/FunKySchEmaName/FUNCTION_THAT_YELLS_AT_YOU
```

This would not work due to the way casing works in PostgreSQL. You would need to quote the `"FunKySchEmaName"."FUNCTION_THAT_YELLS_AT_YOU"` for those to be picked up.

That said, can you add a unit test for this both for this parsing logic and adjust the tests to use funky casing in the function name so this gets tested permanently?

```suggestion
		e.HookName = fmt.Sprintf("%q.%q", schema, table)
```